test/src/main/configurations/JDBC/ConfigurationResultSetIteratingPipe.xml

```diff
@@ -0,0 +1,56 @@
+<module>
+	<adapter name="ResultSetIteratingPipeTest" 
+		description="Get unprocessed from database and send each record to child adapter">
+		<receiver>
+			<listener className="nl.nn.adapterframework.receivers.JavaListener" serviceName="ibis4test-ResultSetIteratingPipeWithDBcall" />
+		</receiver>
+
+		<pipeline firstPipe="Get records to be processed from database and iterate">
+			<exits>
+				<exit path="READY" state="success"/>
+			</exits>
+
+			<pipe name="Get records to be processed from database and iterate" 
+				className="nl.nn.adapterframework.jdbc.ResultSetIteratingPipe" 
+				query="SELECT * FROM IBISTEMP WHERE TCHAR = 'I' AND TNUMBER = '1' ORDER BY TKEY" 
+				styleSheetName="xsl/CreateMessage.xsl" 
+				collectResults="false"
+				ignoreExceptions="true">
+				<sender className="nl.nn.adapterframework.senders.IbisLocalSender" 
+					javaListener="ResultSetIteratingPipeChildListener" />
+				<forward name="success" path="READY"/>
+			</pipe>
+		</pipeline>
+	</adapter>
+
+	<adapter name="ResultSetIteratingPipeChild" description="Convert raw xml and update db">
+		<receiver>
+			<listener name="ResultSetIteratingPipeChildListener" className="nl.nn.adapterframework.receivers.JavaListener"/>
+		</receiver>
+
+		<pipeline firstPipe="Echo message" transactionAttribute="Required" >
```

Comment:
Misschien goed om hier in een comment bij te zetten bij welke combinaties van transactionAttribute het probleem gezien werd.